### DISCUSSION TITLE
1.16.0 Introduced a bug for laravel 10.x

### DISCUSSION THREAD

**Discussion by francoisauclair911** [spatie/laravel-json-api-paginate#81] [Bugs]
Here is a reproduction repo
https://github.com/francoisauclair911/json-paginate-bug-reproduction

Just run `php artisan serve` and see the difference in response between `jsonPaginate()` and normal `paginate()`

I can confirm this bug isn't triggered in Laravel `11.x`

**Comment by sebj54** [comment]
Hi @francoisauclair911. Unfortunately, I cloned your project, ran `composer install`, `php artisan serve` and the homepage is returning a 500 error:

![image](https://github.com/user-attachments/assets/2709a2a1-f874-4199-98e5-ba3164d23032)

What did I do wrong?

**Reply by francoisauclair911** [reply]
@sebj54 Weird... actually I forgot, could you run

 `php artisan migrate` and `php artisan db:seed`  ?

**Reply by sebj54** [reply]
<img width="1409" alt="image" src="https://github.com/user-attachments/assets/99384050-7671-45db-828b-b52f0baddb3a" />

**Comment by francoisauclair911** [comment]
Steps to run the repro repository  @sebj54 

1. `git clone git@github.com:francoisauclair911/json-paginate-bug-reproduction.git`
2. `cd json-paginate-bug-reproduction`
3. `composer install`
4. `cp .env.example .env`
5. `php artisan key:generate`
6. `php artisan migrate`
7. `php artisan db:seed`
8. `php artisan serve`

**Comment by AlexVanderbist** [comment]
Hi, I've re-opened the issues section. Please create an issue there if the issue persists. Feel free to also include the different outputs in that issue directly. Thanks!
